app/views/devise/two_step_verification_session/new.html.erb

```diff
@@ -23,6 +23,15 @@
           autocomplete: 'off',
           'aria-describedby': 'code-hint' %>
       </div>
+      <div data-module="toggle" class="if-no-js-hide">
```

Comment:
Almost missed this one. The `if-no-js-hide` should be on the link below. If there's no JS we want to show the text without the toggle.